services/models/pod.go

```diff
@@ -1,6 +1,8 @@
 package models
 
 import (
+	"encoding/json"
+
```

Comment:
is there a need a space between imports?

Reply:
standard, vendors and finally kiali ones, if @jmazzitelli didn't change it. :)
But I admit sometimes I forget them.